src/Map.jsx

```diff
@@ -14,8 +14,45 @@ import {
   getVehicleMarkersLayer,
   getSubRoutesLayer,
 } from './Route';
+import muniRoutesGeoJson from './res/muniRoutes.geo.json';
+import Checkbox from './Checkbox';
 import ControlPanel from './ControlPanel';
 
+const notAlpha = /[^a-zA-Z]/g;
```

Comment:
I'm pretty sure you didn't write the code in this file. Could you run `git checkout origin/master -- src/Map.jsx` and then amend your previous commit with `git commit -a --amend`?